lib/Minz/View.php

```diff
@@ -258,29 +242,22 @@ public static function headScript () {
 				$scripts .= ' async="async"';
 			}
 			$scripts .= '></script>';
-
-			if ($cond) {
-				$scripts .= '<![endif]-->';
-			}
-
 			$scripts .= "\n";
 		}
 
 		return $scripts;
 	}
-	public static function prependScript ($url, $cond = false, $defer = true, $async = true, $id = '') {
+	public static function prependScript ($url, $defer = true, $async = true, $id = '') {
```

Comment:
It is a bit dangerous to change the signature definition of a public method of this type. It does not look like we use the additional parameters in the core of FreshRSS, but it could be good to double-check some of our known extensions